snapcraft/internal/states/_state.py

```diff
@@ -81,6 +86,11 @@ def _get_differing_keys(dict1, dict2):
     return differing_keys
 
 
+def get_global_state():
+    with open(os.path.join('snap', '.snapcraft', 'state'), 'r') as state_file:
```

Comment:
@sergiusens done: https://github.com/snapcore/snapcraft/pull/1340/commits/3c28849d0d1c974843d1d684f9502af9e6b6bd07
I couldn't find better locations, so please check if they make sense to you.